composer.json

```diff
@@ -9,17 +9,18 @@
         }
     },
     "require": {
-        "php": ">=7.0",
-        "symfony/dependency-injection": "~2.8|~3.0",
-        "symfony/event-dispatcher": "~2.8|~3.0",
-        "symfony/http-kernel": "~2.8|~3.0",
-        "symfony/stopwatch": "~2.8|~3.0",
-        "phpro/soap-client": "~0.2"
+        "php": ">=7.3",
+        "symfony/dependency-injection": "~3.0|~4.0",
```

Comment:
I tried with 5.0, but things get fucky because of other dependencies of the project, especially PHPUnit. 

also, I wanted to keep things step by step and be inclusive. rather than jumping over a version of Symfony that is still quite common.

Reply:
Feel free to upgrade phpunit to the latest as well if that solves the issue